src/gt4py/definitions.py

```diff
@@ -21,22 +21,20 @@
 import numbers
 import operator
 from dataclasses import dataclass
-from typing import Mapping, Optional, Sequence, Tuple
+from typing import Tuple
 
 import numpy
 
 from gt4py import utils as gt_utils
 from gt4py.utils.attrib import Any, AttributeClassLike
 from gt4py.utils.attrib import Dict as DictOf
-from gt4py.utils.attrib import List as ListOf
-from gt4py.utils.attrib import Tuple as TupleOf
 from gt4py.utils.attrib import attribclass, attribkwclass, attribute
 
 
 class CartesianSpace:
     @enum.unique
     class Axis(enum.Enum):
-        I = 0
+        I = 0  # noqa: E741
```

Comment:
I suggest to add a short description of QA error codes, specially for the not-so-common ones concrete 
```suggestion
        I = 0  # noqa: E741  # Do not use variables named 'I', 'O', or 'l' 
```